### PR TITLE
BXMSDOC-5389: Draft of "Developing decision services" doc for Kogito.

### PR DESCRIPTION
**Pull request by sterobin**
**Michael/Guilherme and Matteo (Eder if needed)**, can you review the [DMN portion](http://file.rdu.redhat.com/~sterobin/BXMSDOC-5389/#designing_a_decision_service_using_dmn_models) of the new "Developing decision services" doc for Kogito Dev Preview? It's in near-final state, except a few cross-ref links coming soon. So this is essentially how the DMN content will be delivered for Dev Preview, pending your feedback. i.e., I won't be submitting for another review unless the DMN tooling UI changes in 0.8-0.9.

**Edoardo, Mario**: The DRL, XLS, and decision engine portions are for you, but don't review yet because Edoardo is planning to go through on his end after I merge this initial PR for some substantial changes throughout. So postponing your review of those sections until Edoardo is done.

Links:
* [JIRA](https://issues.redhat.com/browse/BXMSDOC-5376)
* [Enterprise doc preview](http://file.rdu.redhat.com/~sterobin/BXMSDOC-5389/)
* [Community doc preview](http://file.rdu.redhat.com/~sterobin/BXMSDOC-5389_COMM/#chap_kogito-developing-decision-services) - Same, but conditioned and with additional content. Currently shows BA versioning (7.34). Can't change yet.

### CODE REVIEW

File: doc-content/kogito-docs/src/main/asciidoc/dmn/con_dmn-documentation.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[id='con_dmn-documentation_{context}']
+= DMN model documentation in the {PRODUCT} DMN designer
+
+In the {PRODUCT} DMN designer, you can use the *Documentation* tab to generate a report of your DMN model that you can print or download as an HTML file for offline use. The DMN model report contains all decision requirements diagrams (DRDs), data types, and boxed expressions in your DMN model. You can use this report to share your DMN model details or as part of your internal reporting workflow.
```
Comment (manstis) [inline doc-content/kogito-docs/src/main/asciidoc/dmn/con_dmn-documentation.adoc]: `kogito` does not support printing or downloading the documentation (or, more specifically, they do not exist in the VSCode form of DMN designer. The buttons still exist in both the _Github_ and _online_ incarnations of the editor).

See https://issues.redhat.com/browse/KOGITO-1156

**Review by manstis — COMMENTED** [review]
Hi @sterobin just a minor comment about the DMN editor documentation tab capabilities in `kogtio` otherwise everything seems good. I noticed you've included images for _imported models_ but appear to have commented out their use in the applicable `.adoc` files (along with other comments regarding the current state of imported models in `kogito` in various documents. Good stuff).

File: doc-content/kogito-docs/src/main/asciidoc/dmn/con_dmn-documentation.adoc (hunk 1 above)
Comment (sterobin) [reply doc-content/kogito-docs/src/main/asciidoc/dmn/con_dmn-documentation.adoc]: @manstis, oops! I saw that change in the latest UI and updated the screenshot but forgot to change the text. :) Good catch. Fixed. Thank you!

**Comment by karreiro** [comment]
@sterobin I just noticed you closed this PR, should I keep reviewing it?

Anyhow, I've noticed some minor details. Kogito does not have the **Overview** tab and does not have the **Included models** feature enabled, so I think these images need to be updated:
  - `_images/kogito/dmn/dmn-custom-datatypes-tab.png`
  - `_images/kogito/dmn/dmn-designer-nav-view2.png`
  - `_images/kogito/dmn/dmn-include-model-data-types.png`
  - `_images/kogito/dmn/dmn-include-model-data-types.png `
  - `_images/kogito/dmn/dmn-include-model-drd.png`
  - `_images/kogito/dmn/dmn-include-model-expression-pmml.png`
  - `_images/kogito/dmn/dmn-include-model-list-pmml.png `
  - `_images/kogito/dmn/dmn-include-model-list.png `
  - `_images/kogito/dmn/dmn-include-model-pmml.png`
  - `_images/kogito/dmn/dmn-include-model-pmml.png `
  - `_images/kogito/dmn/dmn-include-model.png`
  - `_images/kogito/dmn/dmn-tDriver-custom-datatype.png`
  - `_images/kogito/dmn/dmn-tFine-custom-datatype.png`
  - `_images/kogito/dmn/dmn-traffic-violations-drd.png`
  - `_images/kogito/dmn/dmn-tViolation-custom-datatype.png`

**Pull request closed without merging**